tket/src/Transformations/Rebase.cpp

```diff
@@ -118,10 +118,18 @@ Transform Transform::rebase_tket() {
 Circuit Transform::tk1_to_PhasedXRz(
     const Expr& alpha, const Expr& beta, const Expr& gamma) {
   Circuit c(1);
-  std::vector<Expr> px_params = {beta, alpha};
-  c.add_op<unsigned>(OpType::Rz, alpha + gamma, {0});
-  if (!equiv_0(beta, 4)) {
-    c.add_op<unsigned>(OpType::PhasedX, px_params, {0});
+  if (equiv_expr(beta, 1)) {
+    // Angles β ∈ {π, 3π}
+    c.add_op<unsigned>(OpType::PhasedX, {beta, (alpha - gamma) / 2.}, {0});
+  } else if (equiv_expr(beta, 2, 4)) {
+    // Angle β = 2π
+    c.add_op<unsigned>(OpType::Rz, alpha + gamma + 2, {0});
+  } else if (equiv_0(beta, 4)) {
+    // Angle β = 0
+    c.add_op<unsigned>(OpType::Rz, alpha + gamma, {0});
```

Comment:
```suggestion
  } else if (equiv_expr(beta, 0)) {
    // Angle β ∈ {0, 2π}
    c.add_op<unsigned>(OpType::Rz, alpha + beta + gamma, {0});
```
I think we can combine these cases into one like this?